### ISSUE TITLE
Joss paper review

### ISSUE THREAD

**Issue by KBodolai** [yotarazona/scikit-eo#21]
Hi, reviewing `paper.md` for openjournals/joss-reviews#6692  --

First of all, great job, I think some of the abstractions presented here are great, and do a very good job at lowering the barrier to use ML models with EO data.

In my work I routinely process large volumes of remote sensing data and use ML models for a variety of applications, so hopefully my experience will be useful here, a few points:

My main concern is about the claim of scalability - a lot of it seems to depend on rasterio datasets and local processing - I struggle to see how this can work for very large areas, where datasets quickly become larger-than-memory. Particularly at the inference stage of the ML process - where we predict using the trained model for the whole area. It can be made scalable, but I don't think it is scalable by default.

## Summary
- In the third paragraph of the summary you mention that `scikit-eo` is centralized and scalable. What does centralized in this context mean? as I mentioned in #19 , it would be great to add an example to contextualize the 'scalable' part. 

### scikit-eo as a research tool
- I think there's a typo in the second-to-last paragraph, where it says softmaxis instead of softmax. 
- In the same paragraph it says that linreg models and decision trees don't involve multiple layers of abstraction, what does this statement mean? 

## Functionalities
- In the table there's a typo for `sma` -> classification sup-pixel instead of sub-pixel. 
- Out of curiosity, what does the r in front of kmeans in `rkmeans` stand for?
- The functionality list seems a bit arbitrary to me - for instance, you have implemented atmospheric correction for multispectral data, but it seems to only work for Landsat. Also there could be a lot of 'preprocessing' steps done to SAR data (e.g. before the fusion step, such as multilooking). Is there a plan for this to be expanded in the future?  
- (To be fair, I think it's a very ambitious thing to try to encompass all preprocessing we do with RS data in a package, particularly when dealing with different sources of data such as SAR and multispectral data. For instance )

## State of the field
I think there's some missing citations here of software that is aimed to do analysis with remote sensing data, namely:
- [SITS](https://github.com/e-sensing/sits) -> particularly for time series
- [torchgeo](https://github.com/microsoft/torchgeo) ->  provides a lot of functionality around using deep learning to geospatial data.

**Comment by mfbenitezp** [comment]
Thanks again for your meaningful comments and suggestions; I have addressed the most recent suggestions and left some comments addressing some of the questions included. 

1. The r in front of kmeans in rkmeans stands for remote, it is just a name we provide to avoid confusion with other functions. 
2. We added the clarification, and yes, we plan to keep extending the package once we get the final feedback from users.
3. We have adjusted the initial paragraph for clarity. 
4. We have added the suggested packages

**Comment by yotarazona** [comment]
Thank you very much @KBodolai  for all your comments. Most of them were addressed, now I will address some other missing issues.

**Scalability:**
Indeed, this is a very important topic but it was already clarified in issue #19 . Please visit this discussion for more details.

**Atmospheric correction**:
This topic is tough because for now only Landsat images can be atmospherically corrected. Honestly, I don't think I'll develop corrections for other images because it's a little outside my area of ​​research and knowledge. The implemented atmospheric correction method is the simplest image-based method but widely used in the community.

**Future versions:**
The idea we have of this package is to try to insert all the preprocessing and processing tools for satellite images and make the analyzes more user-friendly. All the tools presented here were developed out of need for this user, and I still have some processes that I plan to implement in this package. However, other developers/researchers will be invited to implement other new tools to process data not only optical, but radar, for example. Then, God willing, we will see what the future brings in the medium or long term.

Thank you very much again for your comments/suggestions that improved the ```scikit-eo``` package.

Best,
Yonatan

**Comment by KBodolai** [comment]
Thank you both for addressing this thoroughly!
